addon/mixins/sortable-item.js

```diff
@@ -431,17 +438,16 @@ export default Mixin.create({
     @return Promise
   */
   _waitForTransition() {
-    return new Promise(resolve => {
-      run.next(() => {
-        let duration = 0;
-
-        if (this.get('isAnimated')) {
-          duration = this.get('transitionDuration');
-        }
-
-        run.later(this, resolve, duration);
+    if (this.get('isAnimated')) {
+      return new Promise(resolve => {
+        run.next(() => {
+          let duration = this.get('transitionDuration');
+          run.later(this, resolve, duration);
+        });
       });
-    });
+    } else {
+      return Promise.resolve();
+    }
```

Comment:
Yes, without this the integration "unit test" wasn't working, because that kind of test doesn't actually wait for promises to solve. I think I'm free to remove this now that we switched to a proper acceptance test.